src/main/java/frc/robot/hardware/phoenix6/PhoenixProUtils.java

```diff
@@ -11,13 +11,13 @@ public static <T> StatusSignal<T> getRefreshedSignal(boolean refresh, StatusSign
 		return refresh ? signal.refresh() : signal;
 	}
 
-	public static boolean checkWithRetry(Supplier<StatusCode> statusCodeSupplier, int numberOfTries) {
-		for (int i = 0; i < numberOfTries; i++) {
+	public static StatusCode checkWithRetry(Supplier<StatusCode> statusCodeSupplier, int numberOfTries) {
+		for (int i = 0; i < numberOfTries - 1; i++) {
```

Comment:
does this mean that for every robot loop we use the canbus 5 times for each device on canbus?

Reply:
yes i think, its for config and operations like it that we want them to surely happen

Reply:
Sounds weird, if you have canbus signal loss that’s not the solution..

Reply:
the solution is to connect the bus better, but thats a safety mechanism for that